shared/src/bundles/workouts/validation-schemas/user-workouts.validation-schema.ts

```diff
@@ -0,0 +1,27 @@
+import { z } from 'zod';
+
+import { Activity } from '../../../enums/enums.js';
+
+type UserWorkoutsPUTRequestValidationDto = {
```

Comment:
```suggestion
type WorkoutsUpdateRequestValidationDto= {
```
I think it is more clear